### PR TITLE
Fix Docker Build

### PR DESCRIPTION
**Pull request by 0ortmann**
- Fix Docker build
- Update instructions in `INSTALLATION.md`
- Update `./scripts/docker-build`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,73 +1,55 @@
...
+
+
+# Production image: copy application
 FROM debian:buster-slim
-LABEL maintainer="tobias.mayer@tenzir.com"
-LABEL builder=false
+LABEL maintainer="engineering@tenzir.com"
 
 ENV PREFIX /usr/local
 ENV LD_LIBRARY_PATH $PREFIX/lib
```
Comment (dominiklohmann) [inline Dockerfile]: Why is this necessary?

File: Dockerfile [hunk 2]
```diff
@@ -1,75 +1,57 @@
...
-      libc++1 \
-      libc++abi1 \
-      libpcap0.8 \
-      openssl && \
-    echo "Adding tenzir user" && \
-    groupadd --gid 20354 tenzir && useradd --system --uid 20354 --gid tenzir tenzir
+RUN apt-get -qq update && apt-get -qq install -y libc++1 libc++abi1 libpcap0.8 \
+  openssl
+RUN echo "Adding tenzir user" && \
+  groupadd --gid 20354 tenzir && useradd --system --uid 20354 --gid tenzir tenzir
```
Comment (mavam) [inline Dockerfile]:
```suggestion
  groupadd --gid 20097 tenzir && useradd --system --uid 20097 --gid tenzir tenzir
```

File: INSTALLATION.md [hunk 3]
```diff
@@ -55,7 +55,26 @@ TODO
...
+branch or tag.
+
+```sh
+./scripts/docker-build <BRANCH>
+```
+
+Start VAST in a container and detach it to the background.
+
+```sh
+docker run -t --name=vast --rm -v /var/log/vast:/data -p 42000:42000 tenzir/vast:latest start
```
Comment (mavam) [inline INSTALLATION.md]: Should this maybe be `/var/db/vast` instead of `log`?
